client-react/src/pages/app/app-settings/AppSettingsDataLoader.tsx

```diff
@@ -400,6 +404,31 @@ const AppSettingsDataLoader: React.FC<AppSettingsDataLoaderProps> = props => {
     return true;
   };
 
+  const errorPagesUpdated = (current: FormErrorPage[], origin: FormErrorPage[] | undefined) => {
+    let success = true;
+    current.forEach(async errorPage => {
+      if (errorPage.content) {
+        const response = await SiteService.AddOrUpdateCustomErrorPageForSite(resourceId, errorPage.errorCode, errorPage.content);
+        success = response.metadata.success;
+        if (!success) {
+          return success;
+        }
+      }
+    });
+
+    origin?.forEach(async errorPage => {
+      const index = current.findIndex(x => x.key == errorPage.key);
+      if (index < 0) {
+        const response = await deleteCustomErrorPageForSite(resourceId, errorPage.errorCode);
+        success = response.metadata.success;
+        if (!success) {
+          return success;
+        }
```

Comment:
If the intent is to have `success` be set to `false` if any deletes fail:

```suggestion
        success = success && response.metadata.success;
```